internal/boxcli/shell_test.go

```diff
@@ -14,6 +14,9 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.jetpack.io/devbox/internal/testframework"
 )
 
 func testShellHello(t *testing.T, name string) {
```

Comment:
@gcurtis These tests fail when I try them. Do you have a recommendation on how to update them? or should we just remove?